msi/release.sh

```diff
@@ -118,10 +118,35 @@ SCRIPT_PATH="$(cd "$(dirname "$0")" && pwd)"
 cd "$SCRIPT_PATH" ||
 die "Could not switch directory to $SCRIPT_PATH"
 
+# Generate the ReleaseNotes.html file
+test -f ../ReleaseNotes.html &&
+test ../ReleaseNotes.html -nt ../ReleaseNotes.md &&
+test ../ReleaseNotes.html -nt release.sh || {
+    # Install markdown
+    type markdown ||
+    pacman -Sy --noconfirm markdown ||
+    die "Could not install markdown"
+
+    (printf '%s\n%s\n%s\n%s %s\n%s %s\n%s\n%s\n%s\n' \
+        '<!DOCTYPE html>' \
+        '<html>' \
+        '<head>' \
+        '<meta http-equiv="Content-Type" content="text/html;' \
+        'charset=UTF-8">' \
+        '<link rel="stylesheet"' \
+        ' href="usr/share/git/ReleaseNotes.css">' \
+        '</head>' \
+        '<body class="details">' \
+        '<div class="content">'
+     markdown ../ReleaseNotes.md ||
+     die "Could not generate ReleaseNotes.html"
+     printf '</div>\n</body>\n</html>\n') >../ReleaseNotes.html
+}
+
```

Comment:
So, the NuGet package requires the CSS file to be in a different location. Since generating this file is not a big deal, I'm going with the "OR" option--i.e. always regenerate the file, and add the parameter to the `render-release-notes.sh` script to specify the relative location of the CSS file to the HTML file. It'll be up to the installer `release` scripts to ensure the CSS exists in the right location.